scripts/module-name/index.js

```diff
@@ -0,0 +1,11 @@
+// Script examples for ScriptAPI
+// Author: defowler2005#4812 <Jayly Discord>
```

Comment:
You cannot create a new package named "module-name", it's not descriptive for what you trying to do.